mscs

```diff
@@ -24,12 +24,9 @@ LOCATION="/opt/mscs"
 # Setup the default location of the mscs.defaults file.
 MSCS_DEFAULTS="$LOCATION/mscs.defaults"
 
-# Setup the arguments to the msctl script.
-MSCS_ARGS="-p $PROG -l $LOCATION -c $MSCS_DEFAULTS $@"
-
 # Run the msctl script.
 if [ "$USER_NAME" = "$(whoami)" ]; then
-  msctl $MSCS_ARGS
+  msctl -p "$PROG" -l "$LOCATION" -c "$MSCS_DEFAULTS" "$*"
```

Comment:
did some testing before and after this commit, and i think this commit breaks the script--it fails to recognize command input  (try typing "mscs create test" and see what happens). i think this change is what causes the issue

Reply:
commit 12aaf40

Reply:
I had noticed this a while ago, but forgot to push the patch. It has now been pushed.